google/cloud/storage/client.py

```diff
@@ -1761,7 +1762,10 @@ def generate_signed_post_policy_v4(
         elif bucket_bound_hostname:
             url = f"{_bucket_bound_hostname_url(bucket_bound_hostname, scheme)}/"
         else:
-            url = f"https://storage.googleapis.com/{bucket_name}/"
+            api_access_endpoint = os.getenv(
+                "API_ENDPOINT_OVERRIDE", "https://storage.googleapis.com"
```

Comment:
I know we were having trouble with some of the signing tests; does this change make any tests pass that weren't previously passing?

Reply:
Let's skip this as discussed because the if/elif is not covered by system tests yet.